using-timescaledb/continuous-aggregates.md

```diff
@@ -151,6 +151,13 @@ will be spawned immediately following completion. This means that the part of th
 data processed by previous jobs will be available even before the
 materialization is fully caught up.
 
+The `timescaledb.ignore_invalidation_older_than` parameter control how
+adding records with an old timestamp will trigger update of the
+continuous aggregate. An update that is older than this parameter will
+not trigger an update so, for example, setting the parameter to "1
+month" will ensure that any update for a record that is older than 1
+month will not trigger an update of the view.
+
```

Comment:
```suggestion
By default, all invalidations are processed. When ignore_invalidation_older_than is specified, the view is freed from the underlying data, thus allowing those data points to exist as they are in the continuous aggregate without worrying about changes to the data invalidating the view. As a result, refresh jobs will complete faster since they are only dealing with new data as part of the refresh, and you will also have the option to implement a data retention policy. <link to data retention policy section>
```
(adapted from blog material).

Reply:
This is incredibly hard to read:

> By default, all invalidations are processed. When `ignore_invalidation_older_than` is specified, the view is freed from the underlying data, thus allowing those data points to exist as they are in the continuous aggregate without worrying about changes to the data invalidating the view.

What does it mean to be "freed from the underlying data"? What does it mean for a data point to "exist as they are"? It is not clear what an "invalidation" is, it is not described earlier in the text.

Going to try with some other rewrite.

Reply:
For the second part:

> As a result, refresh jobs will complete faster since they are only dealing with new data as part of the refresh, and you will also have the option to implement a data retention policy. <link to data retention policy section>

What does it mean to have the "option to implement a data retention policy"? Can you not use that if you do not set a value for `ignore_invalidation_older_than`, or is it just that it does not make sense?

Reply:
Rewrote to something. PTAL.

Reply:
Rewritten and some clarifications in other threads. Resolving.